src/BenchmarksClient/Workers/SignalRWorker.cs

```diff
@@ -147,10 +159,10 @@ public SignalRWorker(ClientJob job)
             }
 
             cts.Token.WaitHandle.WaitOne();
-            await StopAsync();
+            await StopJobAsync();
```

Comment:
`Stop` is called from both the Client service and the worker?

Reply:
Hm, this was the previous implementation.

Reply:
Actually this is handled by a check in Stopped. There is a stopped flag that is checked to make sure that we don't call stop more than once but your comment also made me realize that we need to flip this flag back every time start is called.